.gitignore

```diff
@@ -21,3 +21,4 @@
 .packer_plugins
 packer_cache
 packer-builder-arm
```

Comment:
If changing name of the package, should this be kept?

Reply:
> If changing name of the package, should this be kept?

`packer-builder-arm` will not be required anymore I guess.